CalibTracker/SiPixelESProducers/plugins/SiPixel2DTemplateDBObjectESProducer.cc

```diff
@@ -39,19 +39,9 @@ std::shared_ptr<const SiPixel2DTemplateDBObject> SiPixel2DTemplateDBObjectESProd
 	GlobalPoint center(0.0, 0.0, 0.0);
 	float theMagField = magfield.product()->inTesla(center).mag();
 
-    std::string label = "numerator";   // The correct default
-	//  std::string label = "denominator"; // Outdated. Used for MC in older GT's
-	//  std::string label = "";      // Outdated. Old default 
-
-	if(     theMagField>=-0.1 && theMagField<1.0 ) label = "0T";
-	else if(theMagField>=1.0  && theMagField<2.5 ) label = "2T";
-	else if(theMagField>=2.5  && theMagField<3.25) label = "3T";
-	else if(theMagField>=3.25 && theMagField<3.65) label = "35T";
-	else if(theMagField>=3.9  && theMagField<4.1 ) label = "4T";
-	else {
-		//label = "3.8T";
-		if(theMagField>=4.1 || theMagField<-0.1) edm::LogWarning("UnexpectedMagneticFieldUsingDefaultPixel2DTemplate") << "Magnetic field is " << theMagField;
-	}
+        std::string label = "numerator";   // The correct default
+	if(theMagField>=4.1 || theMagField<-0.1) edm::LogWarning("UnexpectedMagneticFieldUsingDefaultPixel2DTemplate") << "Magnetic field is " << theMagField;
```

Comment:
Should these values be changed to 3.9 and 3.65, respectively, to match the values in 
[PixelCPEClusterRepair.cc#L91](https://github.com/cms-sw/cmssw/blob/db03cb92a0da0c62ee5a88ab20a4d185c60b7a81/RecoLocalTracker/SiPixelRecHits/src/PixelCPEClusterRepair.cc#L91)?

Reply:
I think the numerical limits in the two classes are serving well distinct purposes:
- here an `edm::LogWarning` warning is issued if the magnetic field strength passed to the code is physically "impossible" (`<0` ,` >4`)
- in [PixelCPEClusterRepair.cc#L91](https://github.com/cms-sw/cmssw/blob/db03cb92a0da0c62ee5a88ab20a4d185c60b7a81/RecoLocalTracker/SiPixelRecHits/src/PixelCPEClusterRepair.cc#L91) those are used to decide when to turn off the CluterRepair CPE treatment due to the lack of an appropriate calibration for the not fully ON cases.  

Moreover if your suggestion is applied we'll have a warning from the `ESProducer` every time we collect 0T data (which is a completely normal and necessary data-taking mode).

Reply:
OK, thanks for the explanation. Indeed, I was merely wondering if the two sets of numerical limits serve distinct purposes.